packages/api/src/graphql/article.ts

```diff
@@ -176,17 +178,81 @@ export const GraphQLArticle = new GraphQLObjectType<Article, Context>({
     createdAt: {type: GraphQLNonNull(GraphQLDateTime)},
     modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
 
-    draft: {type: GraphQLArticleRevision},
-    published: {type: GraphQLArticleRevision},
-    pending: {type: GraphQLArticleRevision},
+    draft: {
+      type: GraphQLArticleRevision,
+      resolve: ({id, draft}, _, {urlAdapter}) => {
```

Comment:
Why are you writing the resolver for `darft`, `published`, `pending` and `latest` if you can write it just once in the `GraphQLArticleRevision`?

Reply:
@roosnic1 please check now